cmd/gitserver/server/customfetch_test.go

```diff
@@ -22,6 +22,26 @@ func TestEmptyCustomGitFetch(t *testing.T) {
 	}
 }
 
+func TestDisabledCustomGitFetch(t *testing.T) {
+	mapping := []*schema.CustomGitFetchMapping{
+		{
+			DomainPath: "github.com/foo/normal/one",
+			Fetch:      "echo normal one",
+		},
+	}
+	remoteUrl := "https://8cd1419f4d5c1e0527f2893c9422f1a2a435116d@github.com/foo/normal/one"
+
+	customGitFetch = func() map[string][]string {
+		return buildCustomFetchMappings(mapping)
+	}
+
+	remoteURL, _ := vcs.ParseURL(remoteUrl)
```

Comment:
```suggestion
	remoteURL, err := vcs.ParseURL(remoteUrl)
```
Why are we swallowing the err here?

Reply:
The URLs are statically configured, I think that's why in the original tests it's ignored. Do you think it is better to add `err` handling?